src/epub/text/chapter-10.xhtml

```diff
@@ -80,7 +80,7 @@
 			<p>“Ah! Clem sounds more natural, doesn’t it, Nat?” questioned Cyn laughing; “we knew Clem and C, but <abbr>Mr.</abbr> Stanwood is a stranger!”</p>
 			<p>“Then let us drop him by all means! and now say you are glad to see your old friend!” said Clem, gayly.</p>
 			<p>“We are transported with delight at beholding our Clem, so lately given up as lost forever!” Cyn replied with equal gayety; and Clem, then looking at Nattie, as if he expected her to say something also, she murmured,</p>
-			<p>“I am very glad to meet C,” a remark that sounded cold beside that of enthusiastic Cyn. But in fact Nattie was so confused, so happy, and so strangely timid, that she longed to get away by herself and think it all over and quietly realize it; and besides, in her secret heart, Nattie felt a growing conviction that Cyn used the plural pronoun we more than previous circumstances actually warranted.</p>
+			<p>“I am very glad to meet C,” a remark that sounded cold beside that of enthusiastic Cyn. But in fact Nattie was so confused, so happy, and so strangely timid, that she longed to get away by herself and think it all over and quietly realize it; and besides, in her secret heart, Nattie felt a growing conviction that Cyn used the plural pronoun <i>we</i> more than previous circumstances actually warranted.</p>
```

Comment:
This should be double quotes instead of italics per the manual. Can you do that?